src/main/java/simpaths/model/Person.java

```diff
@@ -844,7 +851,7 @@ private void updateUnemploymentState() {
     //********************************************************
     // method to adjust for one year increment
     //********************************************************
-    private void aging() {
+    void aging() {
```

Comment:
I can't see a reason this would now need to be a public method - was this accidentally commited?

Reply:
```suggestion
    private void aging() {
```